spec/features/claim_journey_does_not_get_cached_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.feature "Claim journey does not get cached", js: true do
+  it "redirects the user to the start of the claim journey if they go back after the claim is completed" do
+    claim = start_student_loans_claim
+    claim.update!(attributes_for(:claim, :submittable))
+    claim.eligibility.update!(attributes_for(:student_loans_eligibility, :eligible))
+    visit claim_path(claim.policy.routing_name, "check-your-answers")
+
+    expect(page).to have_text(claim.first_name)
+
+    click_on "Confirm and send"
+
```

Comment:
I'd perhaps say it would be worth having an assertion here so that we can be confident we're on the final confirmation screen (and thus the session has been cleared) before the back button is hit. Makes the spec a bit safer and more robust because it will fail early with a clear reason why, and also make it less susceptible to false-positives.

Reply:
Good call 👍

Reply:
Done!